### PR TITLE
Get type: BE 

### PR DESCRIPTION
**Pull request by orjwan98**
- [x] Made backend endpoint and set up its handler. 

- [x] Called itemTypes from Airtable and sent them to the FE

### CODE REVIEW

**Review by jema28 — APPROVED** [review]

**Review by amanshawar — APPROVED** [review]

File: server/controllers/getTypes.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const Airtable = require('airtable');
+
+const { Airtable_API_KEY } = process.env;
+if (!process.env.Airtable_API_KEY) {
+  throw new Error('Missing Airtable API KEY env var');
+}
+
+const base = new Airtable({ apiKey: 'keyxrF4iyiMUf7pCB' }).base('appAZnpLnWP0wjAc6');
```
Comment (Hadeel-Salamin) [inline server/controllers/getTypes.js]: why u shared our api key "ya lahwe" :man_facepalming:

**Review by FidaaHersh — APPROVED** [review]

File: server/controllers/getTypes.js (hunk 1 above)
Comment (jema28) [reply server/controllers/getTypes.js]: 😱😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱

Comment (orjwan98) [reply server/controllers/getTypes.js]: @Hadeel-Salamin sharing is caring ;)

Comment (Hadeel-Salamin) [reply server/controllers/getTypes.js]: @orjwan98 hate masareke :p

**Review by jema28 — APPROVED** [review]

File: server/controllers/getTypes.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const base = new Airtable({ apiKey: Airtable_API_KEY }).base('appAZnpLnWP0wjAc6');
+exports.getTypes = (req, res) => {
+  const itemType = [];
+  base('Type').select({
+    view: 'Grid view',
+  }).eachPage((records, fetchNextPage) => {
+    // This function (`page`) will get called for each page of records.
+    records.forEach((record) => {
+      itemType.push({ itemType: record._rawJson.fields.Name, id: record._rawJson.fields.Id });
```
Comment (Hadeel-Salamin) [inline server/controllers/getTypes.js:16]: I would use record.get('Name') and record.get('Id') method instead of going through all the objects in this way

**Review by Hadeel-Salamin — APPROVED** [review]

**Pull request merged**